tools/update_plotlyjs.R

```diff
@@ -65,10 +62,29 @@ withr::with_dir(tmpdir, {
         file.path(lib_dir, "locales", sub("^plotly-locale-", "", basename(locales))),
         overwrite = TRUE
       )
-      # update plot schema
+      # update plot schema and (partial) bundles
       Schema <- jsonlite::fromJSON(Sys.glob("dist/plot-schema.json"))
+      tmp_file <- tempfile(pattern = "plotly_constants_", fileext = ".js")
+      utils::download.file(
+        url = paste0("https://raw.githubusercontent.com/plotly/plotly.js/", basename(zip), "/tasks/util/constants.js"),
+        destfile = tmp_file,
+        quiet = TRUE,
+        mode = "wb"
+      )
```

Comment:
Is this download necessary? Seems like that `tasks/util/constants.js` file should be available locally in the downloaded zip file?